### PR TITLE
Vcenter.py

### PR DESCRIPTION
**Pull request by mabazyar**
This module allows you to retrive all the information of VMs already spun up in vSphere.

### CODE REVIEW

File: sample/Vcenter.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    
+    def diskInfo(self, summary):
+        if not hasattr(summary, 'storage'):
+            return "0"
+        elif not hasattr(summary.storage, "committed"):
+            return "0"
+        return int(summary.storage.committed / 1024**3)   
+    
+    def vmsummary(self, summary, guest):
+
```
Comment (pgbidkar) [inline sample/Vcenter.py]: Please delete this blank line

File: sample/Vcenter.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+                    nics[nic.macAddress] = {}  # Use mac as uniq ID for nic
+                    nics[nic.macAddress]['netlabel'] = nic.network
+                    ipconf = nic.ipConfig.ipAddress
+                    for ip in ipconf:
+                        if ":" not in ip.ipAddress:  # Only grab ipv4 addresses
+                            nics[nic.macAddress]['ip'] = ip.ipAddress
+                            nics[nic.macAddress]['prefix'] = ip.prefixLength
+                            nics[nic.macAddress]['connected'] = nic.connected
+        return nics
+    def diskInfo(self, summary):
```
Comment (pgbidkar) [inline sample/Vcenter.py]: @mabazyar there was blank line in method vmsummary(self, summary, guest) signature and next line in. It is ok to have blank between each method definition.
Sorry if my comment was confusing

**Comment by mabazyar** [comment]
Does it need any additional amendments to be made?
Cheers,

**Comment by pgbidkar** [comment]
@mabazyar Please consider moving this sample to https://github.com/vmware/pyvmomi-community-samples 
Also make sample name more descriptive

**Comment by mabazyar** [comment]
Regarding the sample's name, it is going to be a module needs to be imported to get it worked. This is to be referred as Vcenter to connect to a remote vcenter machine to retrieve all the VMs' entries from.

To me, it would be more succinct enough if it was named as Vcenter. In addition, I added a use case at the preamble of the source code. That being said, I will go with what you would suggest on this .....So, what would you prefer?
Thanks,

**Comment by pgbidkar** [comment]
@mabazyar I suggest please go through the samples in https://github.com/vmware/pyvmomi-community-samples repo. That will give idea about how others have names samples.
There are other samples as well which fetch vcenter details. Please check how different is your sample to other vcenter details sample and depending on that you can name yours

Please move your sample to https://github.com/vmware/pyvmomi-community-samples repo. This repo is specific to pyvmomi.

Thanks

**Pull request closed without merging**